guides/micronaut-oauth2-github/java/src/main/java/example/micronaut/GithubRepo.java

```diff
@@ -1,21 +1,19 @@
 package example.micronaut;
 
+import io.micronaut.core.annotation.Creator;
 import io.micronaut.core.annotation.Introspected;
 
 @Introspected
 public class GithubRepo {
 
-    private String name;
-
-    public GithubRepo() {
+    private final String name;
 
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.